### PR TITLE
(TK-466) Update trapperkeeper to 1.5.6

### PR DESCRIPTION
**Pull request by Magisus**

### CODE REVIEW

**Comment by justinstoller** [comment]
It looks like 1.7.x is missing https://github.com/puppetlabs/clj-parent/commit/de3258f89ef571e915cf9a3494dcaaf8330ce3cb#diff-0fff143854a4f5c0469a3819b978a483 in its history. I think merging this will try to re-build 1.7.5?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## [1.7.5]
```
Comment (justinstoller) [inline CHANGELOG.md]: It looks like 1.7.5 was already tagged with just an update to the clj-ldap dependency and this should be 1.7.6?

Comment (Magisus) [reply CHANGELOG.md]: Oh whoops.

**Comment by Magisus** [comment]
Ah, that's what I get for branching off a tag.

**Comment by Magisus** [comment]
Fixed. I pushed the missing commit directly.

**Comment by justinstoller** [comment]
@jonathannewman wanted to make sure you had visibility into this. Any thoughts or concerns about branching 1.7.x for Irving support?

**Comment by jonathannewman** [comment]
Sounds like a good call to me.  Once this is in, and Irving thaws, I’ll update orchestration services and console services.

**Pull request merged**